Userland/Libraries/LibGfx/Painter.h

```diff
@@ -164,6 +164,8 @@ class Painter {
     DrawOp draw_op() const { return state().draw_op; }
 
     void add_clip_rect(IntRect const& rect);
+    void add_horizontal_clip_range(IntRange const& range);
+    void add_vertical_clip_range(IntRange const& range);
```

Comment:
This seems like something we can already achieve with the existing `add_clip_rect()` and `clip_rect()` APIs.

Reply:
I've changed the code to use `clip_rect()` and `add_clip_rect()` :+1: